### ISSUE TITLE
Math block features are undersold

### ISSUE THREAD

**Issue by johnblommers** [heyman/heynote#47]
Math blocks understand:

- Factorial
- Fractional and whole exponents
- Trigonometric and Transcendental functions
- Understands `pi` and `e`
- Can operate on complex numbers

Non-integers are floating point numbers so you can expect this:

err = 0.1 + 0.2 - 0.3  0.00000000000000005551115123125783

Note that you can click on a computed result and its value goes to the clipboard.

**Comment by heyman** [comment]
Yes, I should highlight more of the Math blocks' features on the website. 

The power of the Math blocks mostly comes from the great [Math.js library](https://mathjs.org/)
